<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PokeBoxes</class>
 <widget class="QFrame" name="PokeBoxes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QFrame" name="selectedFrame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="innerSelectedFrame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,100">
           <item row="0" column="0" colspan="2">
            <layout class="QGridLayout" name="gridLayout" columnminimumwidth="0,130,0">
             <item row="0" column="0" rowspan="3">
              <widget class="QFrame" name="pokemonFrame">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_4">
                <property name="margin">
                 <number>0</number>
                </property>
                <property name="spacing">
                 <number>0</number>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="pokemonSprite">
                  <property name="text">
                   <string>Poke Avatar</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="itemSprite">
                  <property name="text">
                   <string>Item Sprite</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="1" colspan="2">
              <widget class="QLabel" name="boxInfoLabel">
               <property name="text">
                <string>Box #X Slot #X</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="nickNameLabel">
                 <property name="text">
                  <string>Name</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="speciesLabel">
                 <property name="text">
                  <string>Species</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="levelLabel">
               <property name="text">
                <string>Lv. 100</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
               </property>
              </widget>
             </item>
             <item row="2" column="1" colspan="2">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="genderLabel">
                 <property name="text">
                  <string>Gender</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="type1Label">
                 <property name="text">
                  <string>Type1</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="type2Label">
                 <property name="text">
                  <string>Type2</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QGridLayout" name="gridLayout_5" rowstretch="0,100">
             <item row="0" column="0">
              <widget class="QLabel" name="natureLabel">
               <property name="text">
                <string>Nature</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="nature">
               <property name="text">
                <string>Timid</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="movesLabel">
               <property name="text">
                <string>Moves</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="moves">
               <property name="text">
                <string>Move1
Move2
Move3
Move4</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="1">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QPushButton" name="storeButton">
               <property name="text">
                <string>&amp;Store</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="withdrawButton">
               <property name="text">
                <string>&amp;Withdraw</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="switchButton">
               <property name="text">
                <string>Sw&amp;itch</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="deleteButton">
               <property name="text">
                <string>D&amp;elete</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="teamFrame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="PokeButtonsHolder" name="pokemonButtons" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="1">
      <widget class="QPushButton" name="addBox">
       <property name="text">
        <string>&amp;Add new box</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="editBoxName">
       <property name="text">
        <string>&amp;Edit box name...</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="deleteBox">
       <property name="text">
        <string>&amp;Delete current box</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="trainerHome">
       <property name="text">
        <string>&amp;Trainer home</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="6">
      <widget class="QDragReactiveTabWidget" name="boxes">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PokeButtonsHolder</class>
   <extends>QWidget</extends>
   <header>Teambuilder/pokebuttonsholder.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QDragReactiveTabWidget</class>
   <extends>QTabWidget</extends>
   <header>../Utilities/otherwidgets.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
